src/java.base/share/classes/java/security/Security.java

```diff
@@ -162,6 +181,10 @@ private static boolean loadProps(File masterFile, String extraPropFile, boolean
         }
     }
 
+    static Properties getInitialSecurityProperties() {
```

Comment:
Don't think we need this method anymore.

Reply:
Thanks - did another pass through the edits and caught these.  Patch updated.

[Use blessed modifier order in EventNames]


[remove previous edit]